scripts/build_mod_ic.py

```diff
@@ -502,7 +502,20 @@
             "rotate": (0, 0, 0),
         },
     },
-
+    
+    # QFN-8 from DL1636
+    # For PHA-202+
+    "DL1636": {
+        "rows": 2,
+        "pins": 8,
+        "pin_pitch": 1.27,
+        "row_pitch": 5.51,
+        "pad_shape": (1.02, 0.51),
+        "ep_shape": (3.94, 4.11),
+        "chip_shape": (6, 4.90),
+        "pin_shape": (-0.6, 0.42),
```

Comment:
You can also add the EP vias the datasheet recommends: `"ep_vias": (0.5, 1.0, 1.0)` or similar, adjust to fit: (drill, size, gap).

Reply:
Using a gap of 0.5 has given three rows of vias but the footprint datasheet has 2; using 0.8 for the gap instead gives you two rows.